templates/miq-template.yaml

```diff
@@ -446,6 +524,8 @@ objects:
             value: "${POSTGRESQL_MAX_CONNECTIONS}"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: POSTGRESQL_CONFIG_DIR
```

Comment:
@carbonin Consider this custom deployment : 

```
oc new-app --template=manageiq -p POSTGRESQL_IMG_NAME=docker.io/fbladilo/miq-postgresql -p POSTGRESQL_CONFIG_DIR=/etc/blah

oc describe pods postgresql-1-fszjq | grep -A4 miq-pg-config
      /etc/blah from miq-pg-configs (rw)
      /var/lib/pgsql/data from miq-pgdb-volume (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-zlxmm (ro)

sh-4.2$ ls -la /etc/blah
total 8
drwxrwsrwx.  3 root 1000060000   85 Jul 25 16:01 .
drwxr-xr-x. 50 root root       4096 Jul 25 16:01 ..
drwxr-sr-x.  2 root 1000060000   34 Jul 25 16:01 ..7987_25_07_12_01_10.985121115
lrwxrwxrwx.  1 root root         31 Jul 25 16:01 ..data -> ..7987_25_07_12_01_10.985121115
lrwxrwxrwx.  1 root root         28 Jul 25 16:01 01_miq_overrides.conf -> ..data/01_miq_overrides.conf

sh-4.2$ grep include_dir /var/lib/pgsql/data/userdata/postgresql.conf 
#include_dir = 'conf.d'			# include files ending in '.conf' from
include_dir '/etc/blah'

```
Kubernetes grants the configmap volume plugin elevated privileges to do this (assuming the configmap is in place and added into the POD spec). Our default directory creation on image allows older deployments to take advantage of the dependent PR, as they are likely to not have any of the required variables set or configmaps created. It is also a safe default for us to have it created under most circumstances, things can go wrong as I mentioned with configmaps missing or troubled.